<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserDialog</class>
 <widget class="QDialog" name="UserDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>420</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>440</width>
    <height>420</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>440</width>
    <height>420</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add / Edit Account</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>12</number>
   </property>
   <property name="leftMargin">
    <number>28</number>
   </property>
   <property name="topMargin">
    <number>24</number>
   </property>
   <property name="rightMargin">
    <number>28</number>
   </property>
   <property name="bottomMargin">
    <number>24</number>
   </property>
   <item>
    <widget class="QLabel" name="lblTitle">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#2c3e50; margin-bottom:6px;</string>
     </property>
     <property name="text">
      <string>ðŸ‘¤  Account Information</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="horizontalSpacing">
      <number>10</number>
     </property>
     <property name="verticalSpacing">
      <number>10</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="lblFullName">
       <property name="styleSheet">
        <string notr="true">font-weight:bold; font-size:12px;</string>
       </property>
       <property name="text">
        <string>Full Name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditFullName">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>34</height>
        </size>
       </property>
       <property name="placeholderText">
        <string>Enter full name...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblUsername">
       <property name="styleSheet">
        <string notr="true">font-weight:bold; font-size:12px;</string>
       </property>
       <property name="text">
        <string>Username:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditUsername">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>34</height>
        </size>
       </property>
       <property name="placeholderText">
        <string>Letters, numbers, underscores only...</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lblEmail">
       <property name="styleSheet">
        <string notr="true">font-weight:bold; font-size:12px;</string>
       </property>
       <property name="text">
        <string>Email:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEditEmail">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>34</height>
        </size>
       </property>
       <property name="placeholderText">
        <string>example@email.com</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lblPassword">
       <property name="styleSheet">
        <string notr="true">font-weight:bold; font-size:12px;</string>
       </property>
       <property name="text">
        <string>Password:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEditPassword">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>34</height>
        </size>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Minimum 6 characters (leave blank to keep current when editing)...</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="lblRole">
       <property name="styleSheet">
        <string notr="true">font-weight:bold; font-size:12px;</string>
       </property>
       <property name="text">
        <string>Role:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="comboRole">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>34</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>ðŸ‘¤  User</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ðŸ”‘  Admin</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="lblSecQuestion">
       <property name="styleSheet">
        <string notr="true">font-weight:bold; font-size:12px;</string>
       </property>
       <property name="text">
        <string>Security Question:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="lineEditSecQuestion">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>34</height>
        </size>
       </property>
       <property name="placeholderText">
        <string>Security question for password recovery...</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="lblSecAnswer">
       <property name="styleSheet">
        <string notr="true">font-weight:bold; font-size:12px;</string>
       </property>
       <property name="text">
        <string>Answer:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="lineEditSecAnswer">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>34</height>
        </size>
       </property>
       <property name="placeholderText">
        <string>Answer (case-insensitive)...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnSave">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>38</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
    background:#2ecc71;
    color:white;
    border-radius:6px;
    font-size:13px;
    font-weight:bold;
    padding:0 16px;
}
QPushButton:hover{
    background:#27ae60;
}</string>
       </property>
       <property name="text">
        <string>ðŸ’¾ Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>38</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
    background:#ecf0f1;
    color:#7f8c8d;
    border-radius:6px;
    font-size:13px;
    padding:0 16px;
}
QPushButton:hover{
    background:#d5dbdb;
}</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>UserDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>370</x>
     <y>380</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
